src/public/js/timer.js

```diff
@@ -4,20 +4,24 @@ export default (orgs) => {
   document.querySelector('#app').innerHTML = `
     <div>
 
-      <div id="clockdiv"></div>
-      <button id="start"> Start Timer </button>
-      <button id="stop" style="display:none"> Stop Timer </button>
+      <div>
+        <div id="clockdiv"></div>
+        <button id="start"> Start Timer </button>
+        <button id="stop" style="display:none"> Stop Timer </button>
+      </div>
 
       <h4>Organisations</h4>
-      ${
-        orgs.map((org) => {
-          return `
-            <p>${org.id}</p>
-            <p>${org.name}</p>
-            <p>${org.image}</p>
-          `;
-        }).join('')
-      }
+      <div class="columns organisations">
+        ${
+          orgs.map((org) => {
+            return `
+              <div id="${org.id}" class="column organisation">
```

Comment:
having our organisations laid out on a page with 5em divs could take up a lot of real estate on the screen.

I was thinking that we have a [select field](http://bulma.io/documentation/elements/form/) with the organisations as the options within that. And the same for the repos and issues. 

That will keep UI and UX consistent throughout the 3 stages before showing the timer. and also allowing the user to change the organisation/repo/issues easily.